core/src/cgcloud/core/generic_boxes.py

```diff
@@ -286,3 +286,44 @@ def __fix_stupid_locale_problem( self ):
         https://www.banym.de/linux/fedora/problems-with-missing-locale-files-on-fedora-20-made-libvirtd-service-not-starting
         """
         sudo( 'localedef -c -i en_US -f UTF-8 en_US.UTF-8' )
+
+
+class GenericEnvironmentPatcherBox( Box ):
+
+    @classmethod
+    def __patch_etc_environment( cls, env_file, dirs=None, env_pairs=None, envar=None ):
+        r"""
+        >>> f=StringIO('FOO = " BAR " \n  PATH =foo:bar\nBLA="FASEL"')
+        >>> f.seek(0,2) # seek to end as if file was opened with 'a'
+        >>> SparkBox._SparkBox__patch_etc_environment( f, [ "new" ] )
```

Comment:
Bad reference. But see above, when method becomes protected, this name mangling hack is not needed anymore.